src/app/static/templates/login.ftl

```diff
@@ -28,6 +28,12 @@
     </script>
 </head>
 <body>
+    <div class="links text-sm-right">
+        <a class="btn" href="https://reside-ic.github.io/projects/naomi/" target="_blank">About</a>
+        <a class="btn" href="https://naomi.unaids.org/news" target="_blank">News</a>
+        <a class="btn" href="/privacy">Privacy</a>
+        <a class="btn" href="/accessibility">Accessibility</a>
+    </div>
```

Comment:
I don't think this looks aesthetically appealing. Can we put this links somewhere else ? Can we have them as footer ? 

Something like this 
<img width="1364" alt="Screenshot 2023-02-06 at 11 08 18" src="https://user-images.githubusercontent.com/4240748/216956534-2f3d023e-b04f-40cd-9ab5-85ef7641aad2.png">

Reply:
Yeah my first thought was to put them here but then got caught up with trying to make it look good next to all the logos, let me give it another go